components/wb-fieldflow/fieldflow.js

```diff
@@ -472,7 +473,8 @@ var componentName = "wb-fieldflow",
 				defaultselectedlabel: defaultSelectedLabel,
 				lblselector: lblselector,
 				items: itemsToCreate,
-				inline: data.inline
+				inline: data.inline,
+				large: data.large
```

Comment:
Instead of naming the configuration "large" may use `gc-chckbxrdio` or something like that